Python/getversion.c

```diff
@@ -14,7 +14,12 @@ void _Py_InitVersion(void)
         return;
     }
     initialized = 1;
-    PyOS_snprintf(version, sizeof(version), "%.80s (%.80s) %.80s",
+#ifdef Py_GIL_DISABLED
+    const char *buildinfo_format = "%.80s free-threading (%.80s) %.80s";
```

Comment:
Given the message is very visible to people unfamiliar with our build modes it might also be good to prefix have the string read something like "experimental free-threading build" for the 3.13 time being to temper people's expectations.  Once we're exiting the experimental phase in some future release we'd shorten it.

Reply:
Sound reasonalbe to use `experimental free-threading build` except the it is quite long text :) but fine!

Reply:
Wouldn't "experimental free-threading" suffice?

Reply:
I like the longer string for this purpose, it helps emphasize that it is the build rather than the current interpreter mode given that the GIL can be re-enabled both by things like an environment variable and by importing an incompatible extension module.  The _build_ behavior has differences regardless of GIL enablement.